lib/model/node.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * node.js
+ *
+ * Definition for the Node class. A Node is a composable data structure that
+ * contains a collection of Inputs and Outputs, a list of UI styles, and
+ * arbitrary key-value metadata. Nodes can also contain other nodes.
+ */
+
+/**
+ * @constructor Constructor function.
+ * 
+ * @param id - The Node's unique identifier.
+ * @param inputs - An array containing the Node's Inputs.
+ * @param outputs - An array containing the Node's Outputs.
+ * @param styles - An array of strings which contain the styles that will be
+ *                 applied to the node in the visualization.
+ * @param metadata - A standard JS object containing key-value metadata.
+ */
+var Node = function(id, inputs, outputs, styles, metadata) {
+  this._id = id || 0;
+  this._inputs = inputs || [];
+  this._outputs = outputs || [];
+  this._styles = styles || [];
+  this._metadata = metadata || {};
+  this._x = 0;
+  this._y = 0;
+  this._pinned = false;
+};
+
+/**
+ * Getter method for the id property.
+ */
+Node.prototype.getID = function() {
+  return this._id;
+};
+
+/**
+ * @method Getter method for returning an xy-coordinate pair representing the Node's
```

Comment:
I haven't actually run a js doc creator but I'm guessing it's going to interpret "Getter" as the name of the function based on how `@method` is documented
http://usejsdoc.org/tags-function.html